geometry_msgs/PoseStamped object_pose
string object_frame_id